target/start-mailserver.sh

```diff
@@ -49,6 +49,9 @@ DEFAULT_VARS["SPAMASSASSIN_SPAM_TO_INBOX"]="${SPAMASSASSIN_SPAM_TO_INBOX:=0}"
 DEFAULT_VARS["MOVE_SPAM_TO_JUNK"]="${MOVE_SPAM_TO_JUNK:=1}"
 DEFAULT_VARS["VIRUSMAILS_DELETE_DELAY"]="${VIRUSMAILS_DELETE_DELAY:=7}"
 DEFAULT_VARS["NETWORK_INTERFACE"]="${NETWORK_INTERFACE:="eth0"}"
+DEFAULT_VARS["ENABLE_SHARED_INBOX"]="${DOVECOT_ENABLE_INBOX_SHARING:=0}"
+DEFAULT_VARS["NAMESPACE_SEPARATOR_CLAUSE"]="separator = ${DOVECOT_NAMESPACE_SEPARATOR}"
+DEFAULT_VARS["SHARED_INBOX_CONFIG"]="11-shared.conf"
```

Comment:
`DEFAULT_VARS["DOVECOT_SHARED_INBOX_CONFIG"]="11-shared.conf"`

Reply:
Why have you highlighted this one? I have put this into a variable, so the constant `11-shared.conf` doesn't get copy-pasted through the code.

Reply:
>I have put this into a variable, so the constant 11-shared.conf ..

Thats fine. I've just added "Dovecot" to the variable name.

`DEFAULT_VARS["SHARED_INBOX_CONFIG"]="11-shared.conf"` becomes
`DEFAULT_VARS["DOVECOT_SHARED_INBOX_CONFIG"]="11-shared.conf"`